src/main/java/htsjdk/tribble/readers/AsciiLineReader.java

```diff
@@ -36,18 +35,36 @@
  * @author jrobinso
  */
 public class AsciiLineReader implements LineReader, LocationAware {
+    private static final Log log = Log.getInstance(AsciiLineReader.class);
+
     private static final int BUFFER_OVERFLOW_INCREASE_FACTOR = 2;
     private static final byte LINEFEED = (byte) ('\n' & 0xff);
     private static final byte CARRIAGE_RETURN = (byte) ('\r' & 0xff);
 
-    private final PositionalBufferedStream is;
+    private PositionalBufferedStream is;
     private char[] lineBuffer;
     private int lineTerminatorLength = -1;
 
+    protected AsciiLineReader() {};
+
+    /**
+     * TODO: This class implements LocationAware, which requires preservation of virtual file pointers on BGZF inputs.
+     * However, if the inputStream wrapped by this class is a BlockCompressedInputStream, it violates that contract by
+     * wrapping the stream and returning positional file offsets instead.
+     *
+     * @deprecated 8/8/2017 see {@link #from}
```

Comment:
I think the format you want here is `use {@link #from(InputStream)}`. Saying "see" to me sounds like `@see` which has a different usage in javadoc. You can provide the types of the parameters if you want to specify an overridden method.

Also it's not great form to use a date, since the date when the software released is not yet fixed, or easy for a user to determine given a .jar they've got lying around. If you want to include this information, use the version number.